test/integration/len_test.lua

```diff
@@ -0,0 +1,83 @@
+local fio = require('fio')
+
+local t = require('luatest')
+
+local helpers = require('test.helper')
+
+local pgroup = helpers.pgroup.new('len', {
+    engine = {'memtx', 'vinyl'},
+})
+
+pgroup:set_before_all(function(g)
+    g.cluster = helpers.Cluster:new({
+        datadir = fio.tempdir(),
+        server_command = helpers.entrypoint('srv_select'),
+        use_vshard = true,
+        replicasets = helpers.get_test_replicasets(),
+        env = {
+            ['ENGINE'] = g.params.engine,
+        },
+    })
+
+    g.cluster:start()
+
+    g.space_format = g.cluster.servers[2].net_box.space.customers:format()
+end)
+
+pgroup:set_after_all(function(g) helpers.stop_cluster(g.cluster) end)
+
+pgroup:set_before_each(function(g)
+    helpers.truncate_space_on_cluster(g.cluster, 'customers')
+end)
+
+pgroup:add('test_len_non_existent_space', function(g)
+    local result, err = g.cluster.main_server.net_box:call('crud.len', {'non_existent_space'})
+
+    t.assert_equals(result, nil)
+    t.assert_str_contains(err.err, "Space \"non_existent_space\" doesn't exist")
+end)
+
+pgroup:add('test_len', function(g)
+    local bucket_id = 1
+    local other_bucket_id, err = helpers.get_other_storage_bucket_id(g.cluster, bucket_id)
+    t.assert(other_bucket_id ~= nil, err)
```

Comment:
I think it's redundant overcomplication. Let's imagine that sequential ids uniformally distributed across storages.

I think simple loop gives you really high probability that you have at least one tuple on each storage.
```
local customers = {}
for i = 1, 100 do 
    table.insert(customers, {
    id = i, name = tostring(i), last_name = tostring(i), ...
})
end
helpers.insert_objects(g, 'customers', customers)
```

Reply:
But why is the use of a loop more preferable here than in the other tests?(https://github.com/tarantool/crud/blob/master/test/integration/borders_test.lua#L282)

Reply:
I never like such approach to tests - test should be clear and simple as a stick. An attempt to add some "read world sense" is redundant. Moreover currently you need only call "len" and have some guarantees that data is distributed across storages. This loop is shorter than current variant and gives such guarantees.